tests/acceptance/features/webUIProvisioning/groups.feature

```diff
@@ -0,0 +1,78 @@
+@webUI
+Feature: add group
+  As an administrator
+  I want to be able to add, delete and modify groups via the webUI
+  So that I can easily manage groups when user LDAP is enabled
+
+  Background:
+    Given user admin has logged in using the webUI
+    And the administrator has browsed to the users page
+
+  Scenario: Add group
```

Comment:
How is this scenario useful compared to the next scenario?
Did you mean to use a different group name that is not in LDAP?

Reply:
Yes, I just wanted to test that adding simple database group works when LDAP is enabled.